README.md

```diff
@@ -468,11 +462,11 @@ target_link_libraries(your_application_target PRIVATE lib_adapter http_stack_lib
 target_link_libraries(blobs_client_example PRIVATE az_curl CURL::libcurl)
 ```
 
-See the complete cmake file and how to link your own library [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/src/azure/storage/CMakeLists.txt#L26)
+See the complete cmake file and how to link your own library [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/src/azure/iot/CMakeLists.txt#L26)

```

Comment:
nit: Remove the line number. It is already out-of-sync.

```suggestion
See the complete cmake file and how to link your own library [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/src/azure/iot/CMakeLists.txt)
```

If we really want to keep it, it would be line 27:
https://github.com/Azure/azure-sdk-for-c/blob/c30c45efe9f7d5b6f3ae486dbeb121d2fe24e527/sdk/src/azure/iot/CMakeLists.txt#L27